src/monitor/version_compat.h

```diff
@@ -14,11 +14,6 @@
 
 #include "postgres.h"
 
-/* we support Postgres versions 10, 11, 12, 13, 14, 15, and 16. */
-#if (PG_VERSION_NUM < 100000 || PG_VERSION_NUM >= 170000)
-#error "Unknown or unsupported postgresql version"
-#endif
-
```

Comment:
I believe we want to keep that part, that's the only build-time check we have for Postgres compatibility. Just amend it to include Postgres 17.

Reply:
Dropped the removal-commit, re-added build-support for v10.